components/ws-proxy/pkg/proxy/routes.go

```diff
@@ -434,6 +437,11 @@ func installWorkspacePortRoutes(r *mux.Router, config *RouteHandlerConfig, infoP
 				workspacePodPortResolver,
 				withHTTPErrorHandler(showPortNotFoundPage),
 				withXFrameOptionsFilter(),
+				func(h *proxyPassConfig) {
```

Comment:
> The intention of setting it here is that we don't need to verify that the certificate of the workspace internal port is trusted, since they are mostly self-signed certificates

Makes sense, thank you!

I am going to run a brief test in https://github.com/gitpod-io/gitpod/pull/16547, rebuilding that preview environment now (it doesn't exist presently).

Reply:
Looks like the VM got pre-empted...rebuilding :sweat_smile: